src/actions/user.actions.js

```diff
@@ -38,7 +38,11 @@ export const getUserInfoWithToken = ({ token }: { token: string }) =>
     }
   });
 
-export const saveTokenToLocalStorage = ({ token }: { token: string }) => ({
+export const saveTokenToLocalStorage = ({
```

Comment:
If you don't expect more than 1 parameter for an action then you can just have it as a single parameter (no destructuring).

I generally think it's fine to leave out destructuring until I have more than 2 parameters or if I have more than 1 parameter and one of them is a boolean.
since I don't want to see an action like this: myAction(token, true) - which makes me have no idea what true means without going to the action declaration.